src/components/duck.js

```diff
@@ -0,0 +1,53 @@
+AFRAME.registerComponent("duck", {
+  schema: {
+    initialForce: { default: 0 },
+    maxForce: { default: 6.5 },
+    maxScale: { default: 5 }
+  },
+
+  init: function() {
+    this.system = this.el.sceneEl.systems.physics;
```

Comment:
`this.system` is used by aframe for the system for the current component, we should use something different, I suggest `this.physicsSystem`